_dprhtml/js/search.js

```diff
@@ -181,7 +189,7 @@ function finishSearch() {
   DPR_search_mod.hideProgressBar();
   DPR_search_mod.hideCancelButton();
 
-  const searchLink = 'dpr:search?type='+DPR_G.G_searchType+'&query=' + DPR_translit_mod.toVel(DPR_G.G_searchString) + '&MAT=' + DPR_G.G_searchMAT + '&set=' + DPR_G.G_searchSet + '&book=' + DPR_G.G_searchBook.slice(1,-1) + '&part=' + DPR_G.G_searchPart + '&rx=' + DPR_G.G_searchRX;
+  const searchLink = 'dpr:index?feature=search&type='+DPR_G.G_searchType+'&query=' + DPR_translit_mod.toVel(DPR_G.G_searchString) + '&MAT=' + DPR_G.G_searchMAT + '&set=' + DPR_G.G_searchSet + '&book=' + DPR_G.G_searchBook.slice(1,-1) + '&part=' + DPR_G.G_searchPart + '&rx=' + DPR_G.G_searchRX;

```

Comment:
This is the bug fix. I wanted to call it out since there is a lot of test-related setup in this PR.